core/server/lib/members/users.js

```diff
@@ -39,8 +39,10 @@ module.exports = function ({
                     });
                 }));
             }).then((subscriptions) => {
+                const activeSubscriptions = subscriptions.filter(sub => sub.status === 'active');
                 return Object.assign({}, member, {
-                    subscriptions: subscriptions.filter(sub => sub.status === 'active')
+                    subscriptions: activeSubscriptions,
+                    plans: activeSubscriptions.map(sub => sub.plans)
```

Comment:
🤔 Think this should be `.plan` instead of `.plans` in map as we don't have `plans` property on subscription.

```
plans: activeSubscriptions.map(sub => sub.plan)
```

Reply:
On a more general note, and possibly beyond scope of this PR, I feel its possibly better to send `subscriptions` object instead of the stripped down version of `plans` which just includes `Monthly`/`Yearly` as values in an array. `subscriptions` contains a lot more useful information, like the `price` and `validUntil`, which we are not using now but probably gonna be useful to have in context later?

Reply:
Updated :)

Reply:
As for plans vs. subscriptions, I think we want to keep subscription data serverside for now at least. Really plans isn't very well named, but it should be what the application interface with, can be thought of as "roles" in a permission system, whereas the subscription information is for payment backed reasons for why a user has those roles.

Reply:
Yeah that makes sense 👍